deployment/src/main/java/com/github/mcollovati/quarkus/hilla/deployment/HillaTestExtensionProcessor.java

```diff
@@ -66,7 +66,7 @@
 
 class HillaTestExtensionProcessor {
```

Comment:
```suggestion
class QuarkusHillaExtensionProcessor {
```

We should also rename the class, since its not a test anymore.

Reply:
Good point!

Reply:
done